website/docs/tutorials/07-testing/02-cheatcodes/reflect.md

```diff
@@ -0,0 +1,130 @@
+# `reflect`
+```python
+def reflect(self, ids: VmConsts) -> Reflector:
+
+class Reflector:
+    def get(self)
+```
+Loads specified Cairo object into Python type. ```get()``` can return:
+
+- int (for felt)
+- RelocatableValue (for pointer)
+- Cairo Struct (for more complex types)
+
+You can use it to print cairo data and compare complex structures.
+
+```cairo
```

Comment:
what do you think about:
1. Removing `%lang starknet` and other boilerplate which, although, make the snippet executable, is not crucial in expressing the sense
2. Adding a `title="..."` to each snippet as a brief description of what we want to showcase
3. Remove `func test_*()` parts and take following form:
```cairo
struct SimpleStruct:
    member x: felt
end

local simple_struct: SimpleStruct = SimpleStruct(x=10)

%{
        simple_struct = reflect(ids).simple_struct.get()
        print(simple_struct)
        # output:
        # CairoStruct(
        #     x=10
        # )
        assert simple_struct.x == 10 # true
%}
```

?

The goal is to reduce noise in text.

Reply:
I think lack of func test is confusing

Reply:
🤷🏻

Reply:
1. I am whatever, we just should be consistent
2. +1

Reply:
🤷🏻